app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -174,7 +168,7 @@ def parse_env_variables(env_variables)
       def parse_openshift_image(openshift_image)
         id = openshift_image[:dockerImageReference] || openshift_image[:metadata][:name]
         ref = "#{ContainerImage::DOCKER_PULLABLE_PREFIX}#{id}"
-        new_result = parse_container_image(id, ref)
+        new_result = parse_container_image(id, ref).merge!({:image_ref => ref})
```

Comment:
I don't want to create a new object but to change the current one. (That I receive from `@data[:container_images][...]` )